tests/datatypes/test_dict.py

```diff
@@ -229,6 +229,66 @@ def test_method_get(self):
                 print(err)
             """)
 
+    def test_copy(self):
+        self.assertCodeExecution("""
+            x = {42: 'Babel'}
+            y = x.copy()
+            print(y)
+            """)
+
+        self.assertCodeExecution("""
+            x = {42: 'Babel'}
+            y = x.copy()
+            print(x == y)
+            """)
+
+        self.assertCodeExecution("""
```

Comment:
This test and the two above can be combined into a single `self.assertCodeExecution()` call – since they don't don't make any changes to x and y, they shouldn't interfere with each other.